doc/manual/rl-next/git-fetcher.md

```diff
@@ -0,0 +1,18 @@
+---
+synopsis: Git fetching has been reimplemented
+prs:
+  - 9240
+  - 9241
+  - 9258
+  - 9480
+issues:
+  - 5313
+---
+
+Nix can fetch sources from Git natively, during evaluation and locking; outside the sandbox.
```

Comment:
"Natively" is a bit of a vague word. You probably want to say "has builtin support for fetching Git repositories".